### ISSUE TITLE
Grade

### ISSUE THREAD

**Issue by Yuming-Zhang** [Rachelouve/Intro_R_HWM1#2]
Hi everyone!

The total number of points for this HW is 135, and you get 124 at the end with the followings:
(1) 5 points deducted because the repository is not private
(2) 1 point deducted for the wrong name of repository
(3) 2 points deducted for video not available (cannot play) in "introduction"
(4) 1 point deducted for some emojis not able to display
(5) 2 points deducted for some wrong formats of 1st equation and 2nd equation not in-line
 
Your grade is therefore 5.59 out of 6.

**Comment by jetnisa** [comment]
Hi, I don't get why the video doesn't play for you because it actually plays for us.

**Comment by Yuming-Zhang** [comment]
Hi,

Thanks for double check! However, when I open the compiled html file and click to play the video, it shows the video not available as the attached screenshot:


Could you please double check? Thanks a lot!

Best,
Yuming

**Comment by jetnisa** [comment]
Yes I rechecked and it doesn't work anymore, it's very bizarre because it worked 2 weeks ago.
Also, I don't get why you deducted a point for the name of the repository which is Intro_R_hw1 ?

Best, 
jetnisa

**Comment by Yuming-Zhang** [comment]
Hi,

The name of repository is supposed to be Intro_R_hw1 as instructed in the HW1, while yours is Intro_R_HWM1.

Best,
Yuming
